pegasus/sites.v3/code.org/public/educate/csd.md

```diff
@@ -5,7 +5,7 @@ video_player: true
 theme: responsive
 ---
 
-# Our newest course: CS Discoveries #
+# Our newest course: CS Discoveries &nbsp;&nbsp; [<button style="margin-top:0;">View course</button>](https://studio.code.org/courses/csd)
 
 ## <a name="what"></a>What is CS Discoveries?
```

Comment:
Oh I thought you wanted me to add a name to my link. hahah I see this is existing stuff. It's not the best, but I don't want to update this because Josh made these anchor links. I could update site wide but he may already be using these links in his resources. Considering how small a difference the name makes, I don't think it's worth cleaning up now but I can include a tip in our editor guides about choosing more descriptive names for future times.